tests/component/test_trigger_properties.py

```diff
@@ -0,0 +1,115 @@
+"""Contains a collection of pytest tests that validates trigger properties."""
+
+from nidaqmx.constants import TriggerType
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.task import Task
+
+
+def test__trigger__get_float_property__returns_value(ai_voltage_chan_task: Task):
```

Comment:
Rename to ..._returns_default_value. Same for the other tests that validate default values.

Reply:
Renamed to _returns_default_value